Odata-docs/webapi-8/getting-started.md

```diff
@@ -0,0 +1,326 @@
+---
+title:  "Getting Started with ASP.NET Core OData 8"
+description: A getting started guide to creating and running an ASP.NET Core OData 8 application.
+date:   2022-11-11
+ms.date: 11/11/2022
+author: gathogojr
+ms.author: jogathog
+---
+
+# Getting Started with ASP.NET Core OData 8
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v8.md)]
+
+This tutorial shows how to create and run an ASP.NET Core OData 8 application
+
+You'll learn how to:  
+:white_check_mark: Create an ASP.NET Core application  
+:white_check_mark: Add Microsoft.AspNetCore.OData library package  
+:white_check_mark: Define the CLR types  
+:white_check_mark: Add an OData controller  
+:white_check_mark: Build the Edm model and configure the service  
+:white_check_mark: Run the applicaion and interact with the OData service from an API client  
+
```

Comment:
consider adding a table of the API endpoints that will be created by the tutorial. We should highlight the query options enabled by OData. See here for example https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&tabs=visual-studio#overview-1